src/www/app/persistent_server.ts

```diff
@@ -56,6 +59,9 @@ export class PersistentServerRepository implements ServerRepository {
       throw new ServerAlreadyAdded(alreadyAddedServer);
     }
     const server = this.createServer(uuidv4(), serverConfig, this.eventQueue);
```

Comment:
Let's check the `serverConfig.method` instead, before actually creating the server. It's not clear what the side-effects of createServer is. It's better to not have assumptions on that.

Reply:
That's a good point.

_createServer_ is platform specific, so we will need one implementation for each target environment. This was a shortcut to dedup the logic, but I see your concern.

Reply:
Oh, I see that I can just do this in OutlineServer instead.